spec/abstract.md

```diff
@@ -16,15 +16,15 @@ publication mechanism. This offers an interoperability bridge from (or between)
 blockchain ecosystems. Also, without directly supporting environments where the
 web is not practical (e.g., IOT, Lo-Ra, BlueTooth, NFC), the method builds on a
 foundation that fully supports those environments, making future interop of
-identifiers between web and non-web an easy step.
-
+identifiers between web and non-web a manageable step.
+<!-- Nothing is 'easy' in our line of work -->
```

Comment:
I assume we won't include this line, it's just a comment for our review?

Reply:
Yes